### ISSUE TITLE
[Python] Improve test coverage by deleting code that is not in use

### ISSUE THREAD

**Issue by niccokunzmann** [collective/icalendar#641]
Please find **one place** in the source code test coverage that is not in use and delete it.

To close the issue:
- [ ] remove one line that is not tested and not useful
- [ ] create a pull request with your changes and get it merged

What will help:

- Follow the documented [development setup](https://icalendar.readthedocs.io/en/latest/install.html#development-setup)
- After running the tests, you will have a coverage report as an HTML file.
- Talk to us if the line is really useless if you are in doubt.
- Create a Pull Request and we will see the improvements.

Thanks for your help to improve this project!

**Comment by niccokunzmann** [comment]
@harsha-mangena Yes, please! Please create a PR as soon as possible and reference this issue. Looking forward to your contribution. Thanks!

**Comment by niccokunzmann** [comment]
@harsha-mangena How is it going? If you are stuck somewhere, I would like to know this so I can improve the guidance for first-time-contributors, please tell me, thanks!

**Comment by harsha-mangena** [comment]
Hey @niccokunzmann, I am Working on this; I will update you soon. Thank You!
